Makefile

```diff
@@ -1,11 +1,10 @@
 ## Top Level Makefile
 
-## Before making changes here, please take a look at Makeconf
 include ./Makeconf
 
-## 'all' is the default target, i.e. 'make' and 'make all' are the same.
+.DEFAULT_GOAL := all
```

Comment:
Why this change?

Reply:
I generally don't like to depend on the target that lexicaly first appears being the default target.  I like to be explicit about the default target.

Reply:
Well, this is such a commonality that that line is no more than unnecessary noise. It's super easy to detect mistakes in default target.